Source/BridgeComponent.swift

```diff
@@ -50,26 +50,42 @@ open class BridgeComponent: BridgingComponent {
     ///
     /// - Parameter message: The message to be replied with.
     /// - Returns: `true` if the reply was successful, `false` if the bridge is not available.
-    public func reply(with message: Message) -> Bool {
-        return delegate.reply(with: message)
+    public func reply(with message: Message) async throws -> Bool {
+        try await delegate.reply(with: message)
     }
-    
+
+    /// Replies to the web with a received message, optionally replacing its `event` or `jsonData`.
+    ///
+    /// - Parameter message: The message to be replied with.
+    public func reply(with message: Message) {
+        Task { _ = try await delegate.reply(with: message) }
+    }
+
```

Comment:
By adopting async/await we've lost the return value. Most of the times this won't be needed, but I'm wondering if we should provide an optional completion block.

```
    public func reply(with message: Message, 
                      completion: ((Result<Bool, Error>) -> Void)? = nil) {
        Task {
            do {
                let result = try await delegate.reply(with: message)
                completion?(.success((result)))
            } catch {
                completion?(.failure(error))
            }
        }
    }
```

Reply:
Ah, great call. Also, it gets rid of the flaky tests locally!